contracts/courtroom_test.go

```diff
@@ -344,7 +370,7 @@ func TestRegisterFromClient(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Register: expected no error, got %v", err)
 	}
-	backend.Commit()
+	commit(backend)
 
 	registerPlayerCounter, err := swearGame.RegisteredPlayersCounter(&bind.CallOpts{})
```

Comment:
Here we test that the contract does not allow multiple registration for the same client so we need to commit the tx after each registration.